syncback/krsync.sh

```diff
@@ -5,41 +5,60 @@
 
 set -e
 
-# make sure local tmp dir exists
+# make sure local tmp dir exists (see -T= argument in Tiltfile)
 mkdir -p /tmp/rsync.tilt
 
-rsync_path=/bin/rsync.tilt
-tar_path_local=${0%/*}/rsync.tilt.tar  # TODO: option for windows binary
+rsync_tilt_path=/bin/rsync.tilt
+
+install_rsync_tilt() {
+    if [ -z "$RSYNC_PATH" ]; then
+        # rsync already checked/installed
+        return 0
+    fi
+
+    local tar_path_local=${0%/*}/rsync.tilt.tar  # TODO: option for windows binary
+    # ensure our rsync bin is present on container in the expected location
+    set +e # disable exit on error
+    kubectl exec $K8S_OBJECT -- ls "$rsync_tilt_path" > /dev/null 2>&1
```

Comment:
do you need to quote "$K8S_OBJECT" like it was before?

Reply:
Nope! That's part of the fix to #209. It comes into the script as a single argument (quoted from the Tiltfile) but we don't quote it so that kubectl interprets the namespace and container args separately.

Reply:
I'll make a note in the script.